src/odemis/acq/test/move_tfs1_test.py

```
@@ -310,20 +310,20 @@ def test_transformFromSEMToMeteor(self):
 
         # assert that it doesn't raise error when rz is added
         grid1_pos.update(stage_md[model.MD_FAV_SEM_POS_ACTIVE])
         grid2_pos.update(stage_md[model.MD_FAV_SEM_POS_ACTIVE])
 
         # check if no error is raised (test fails if error is raised)
         try:
-            self.posture_manager._transformFromSEMToMeteor(grid1_pos)
-            self.posture_manager._transformFromSEMToMeteor(grid2_pos)
+            self.posture_manager._transform_from_sem_to_fm(grid1_pos)
+            self.posture_manager._transform_from_sem_to_fm(grid2_pos)
         except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)